symbols.xml

```diff
@@ -127,90 +127,90 @@
 
-        <global-address name='created_item_type'/>
-        <global-address name='created_item_subtype'/>
-        <global-address name='created_item_mattype'/>
-        <global-address name='created_item_matindex'/>
-        <global-address name='created_item_count'/>
-        <global-address name='map_renderer'/>
-        <global-address name='d_init'/>
-        <global-address name='enabler'/>
-        <global-address name='flows'/>
-        <global-address name='gps'/>
-        <global-address name='gview'/>
-        <global-address name='init'/>
-        <global-address name='texture'/>
-        <global-address name='timed_events'/>
+        <global-address name='created_item_type'value='0x018b3d1c'/>
+        <global-address name='created_item_subtype'value='0x01cee808'/>
+        <global-address name='created_item_mattype'value='0x018d09ec'/>
+        <global-address name='created_item_matindex'value='0x018ce984'/>
+        <global-address name='created_item_count'value='0x018bc7d8'/>
+        <global-address name='map_renderer'value='0x018bc7f0'/>
+        <global-address name='d_init'value='0x018b3650'/>
+        <global-address name='enabler'value='0x018b33f0'/>
+        <global-address name='flows'value='0x018bae0c'/>
+        <global-address name='gps'value='0x00f930b8'/>
+        <global-address name='gview'value='0x01b2ac90'/>
+        <global-address name='init'value='0x018ce9c0'/>
+        <global-address name='texture'value='0x018ce99c'/>
+        <global-address name='timed_events'value='0x018ce978'/>
         <global-address name='ui' value='0x18b3d28'/>
-        <global-address name='ui_advmode'/>
-        <global-address name='ui_build_selector' value='0x18cfa48'/>
-        <global-address name='ui_building_assign_type'/>
-        <global-address name='ui_building_assign_is_marked'/>
-        <global-address name='ui_building_assign_units'/>
-        <global-address name='ui_building_assign_items'/>
-        <global-address name='ui_look_list' value='0x18ce9a8'/>
-        <global-address name='ui_sidebar_menus' value='0x18bae18'/>
-        <global-address name='world' value='0x18d09f8'/>
+        <global-address name='ui_advmode'value='0x01ce5678'/>
+        <global-address name='ui_build_selector' value='0x018cfa48'/>
+        <global-address name='ui_building_assign_type'value='0x018ce990'/>
+        <global-address name='ui_building_assign_is_marked'value='0x018ce9b4'/>
+        <global-address name='ui_building_assign_units'value='0x018ce96c'/>
+        <global-address name='ui_building_assign_items'value='0x01cee7fc'/>
+        <global-address name='ui_look_list' value='0x018ce9a8'/>
+        <global-address name='ui_sidebar_menus' value='0x018bae18'/>
+        <global-address name='world' value='0x018d09f8'/>
 
         generated cglobals
 
         generated keydisplay
 
-        <global-address name='keydisplay' value='0x018b35fc'/>
+        <global-address name='keydisplay' />
```

Comment:
I just did a bit of analysis (using the "update_keydisplay" function in g_src/enabler_input.cpp) and I've concluded that the actual location of keydisplay is 0x18B3600 - the old address was the very end of an std::string variable (namely, its allocated-length member).

Also keydisplay is not a vector - it is, and always has been, a std::map<InterfaceKey, std::set<std::string,less_sz> >;